app/controllers/catalogues_controller.rb

```diff
@@ -1,7 +1,11 @@
+require 'billing_facade_client'
```

Comment:
Shouldn't have to do this as it should be Rails' job to load everything automatically. If `billing_facade_client` is in the Rails autoload path (either add `lib` to `autoload_paths` in the config, or move `BillingFacadeClient` to `app/lib`) and it'll be loaded automatically.

Reply:
As it is in the /lib folder, I'll remove it because it is already included